src/tests/ftest/dfuse/pil4dfs_fio.yaml

```diff
@@ -0,0 +1,78 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+
+timeout: 600
+
+server_config:
+  name: daos_server
+  engines_per_host: 2
+  engines:
+    0:
+      pinned_numa_node: 0
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
```

Comment:
We only want to run with pmem? And not nvme if available?

Reply:
It is indeed intended as the goal is only to test the client performance test.
@mchaarawi , please could you confirm.

Reply:
From what I know there is at least 4 disks (2 per engines).
However, it seems to not be homogeneous: for sure for the size and for the type of NVMe disks.
@daltonbohning could you confirm this ?

Reply:
The size of disks is not the same for every cluster, but the disks should be homogenous in each cluster. I'm not sure we have a quick reference of each cluster and the specs

Reply:
I would suggest to match what we do for IOR and other io tests. However, i do see in those tests that for some we do SCM only and others we add nvme. so it sounds inconsistent in other tests too.

Reply:
If it is not consistent in the other performance tests , I would suggest to use `storage: auto` as it will be the more robust for future change and compatible with md-on-ssd (from my understanding).
Is it OK for you @mchaarawi and @daltonbohning ?

Reply:
- [ ] Use storage: auto

Reply:
> * [x]  Use storage: auto

Fixed with commit 00a11efcf